substrate/frame/support/src/lib.rs

```diff
@@ -1562,6 +1562,57 @@ pub mod pallet_macros {
 	/// * [`frame_support::derive_impl`].
 	/// * [`#[pallet::no_default]`](`no_default`)
 	/// * [`#[pallet::no_default_bounds]`](`no_default_bounds`)
+	///
+	/// ## Optional: `without_metadata`
+	///
+	/// By default, the associated types of the `Config` trait that require the `TypeInfo` or
+	/// `Parameter` bounds are included in the metadata of the pallet. Please note that the
+	/// `RuntimeEvent` and constants are already included in the metadata.
```

Comment:
```suggestion
	/// `Parameter` bounds are included in the metadata of the pallet.
```

Does not really belong here?